ocs_ci/deployment/terraform.py

```diff
@@ -121,3 +121,24 @@ def destroy_module(self, tfvars, module):
         logger.info(f"Destroying the module: {module}")
         cmd = f"terraform destroy -auto-approve -target={module} '-var-file={tfvars}' '{self.path}'"
         run_cmd(cmd, timeout=1200)
+
+    def change_statefile(self, module, resource_type, resource_name, instance):
+        """
+        Remove the records from the state file so terraform will no longer be
+        tracking the corresponding remote objects.
+
+        Args:
+            tfstate (str): path to terraform.tfstate file
+            module (str): Name of the module
+                e.g: compute_vm, module.control_plane_vm etc.
+            resource_type (str): Resource type
+                e.g: vsphere_virtual_machine, vsphere_compute_cluster etc.
+            resource_name (str): Name of the resource
+                e.g: vm
+            instance (str): Name of the instance
+                e.g: compute-0.j-056vu1cs33l-a.qe.rh-ocs.com
```

Comment:
Should I pass the name `compute-0` or the full name `compute-0.j-056vu1cs33l-a.qe.rh-ocs.com`? How can I retrieve the name above?

Reply:
you have to pass the full name. one can form the full name with below

f"compute-0.{config.ENV_DATA.get('cluster_name')}.{config.ENV_DATA.get('base_domain')}"

Reply:
Okay, I see.